config/locales/en/feedback.yml

```diff
@@ -3,8 +3,9 @@ en:
   feedback:
     new:
       difficulty: 
-        Provider: How easy or difficult was it to apply for legal aid using the new service?
-        Citizen: How easy or difficult was it to use this service?
+        provider: How easy or difficult was it to use this service?
+        citizen: How easy or difficult was it to use this service?
+        unknown: How easy or difficult was it to use this service?
```

Comment:
If all three questions are now the same, can we not just use a single key?
It was only split because there was a need for a different text for applicants and professional users 🤷‍♂️

Reply:
We want to keep the functionality for identifying providers for this ticket https://dsdmoj.atlassian.net/browse/AP-1545.

Reply:
:+1: Sorry if I wasn't clear, I just meant that if the text is the same, do we need to separate the three strings ,provider, citizen and unknown? Wouldn't a single string be okay, e.g.
```ruby
    new:
      difficulty: How easy or difficult was it to use this service?
      done_all_needed: Were you able to do what you needed today?
      satisfaction: Overall, how satisfied were you with this service?
```
The source (:provider, :citizen or :unknown) is still set in the controller at https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/master/app/controllers/feedback_controller.rb#L59

Reply:
👍🏻